tools/clang/include/clang/SPIRV/SpirvBuilder.h

```diff
@@ -896,6 +900,11 @@ void SpirvBuilder::requireCapability(spv::Capability cap, SourceLocation loc) {
   }
 }
 
+bool SpirvBuilder::requiresCapability(spv::Capability cap) {
+  SpirvCapability capability({}, cap);
+  return mod->hasCapability(capability);
+}
```

Comment:
The function does not adds the capability, but checks if it has it already no?
Is the name/comment OK?

Reply:
I renamed the function to `hasCapability`.